src/test/java/com/uid2/admin/vertx/EncryptionKeyServiceTest.java

```diff
@@ -812,6 +813,12 @@ void createKeysetKeysFirstRun() throws Exception {
         // 6 keysets should be created
         assertEquals(6, keysets.keySet().size());
         //Special Keysets are set correctly
+        // Master key site id -1 : keyset id 1
```

Comment:
Hmmm its interesting, I want to say no. Because we should be testing that the const aren't changed but I also want to say yest because it makes the test flakey

Reply:
If we want to test that the consts never change value (do we?), that should be a different test.